File: Tugas coding modul 1/coding c/PRAK110-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>
#include <math.h>

void main(){
    int A, B, C, pythagoras, keliling, luas;
    A = 12;
    C = 5;
    pythagoras = pow(A, 2) + pow(C, 2);
    B = sqrt(pythagoras);
    keliling = (int) (A + B + C);
    luas = (int) (0.5 * C * A);

    printf("Diketahui : \nAlas = %d cm \nTinggi = %d cm \n\n", C, A);
    printf("Jawab : \nSisi A = %d cm \nSisi B = %d cm \nSisi C = %d cm \n", A, B, C);
    printf("Keliling = %d cm \nLuas = %d cm kuadrat", keliling, luas);
}

File: Tugas coding modul 1/coding python/PRAK110-2310817120005-Noviana_Nur_Aisyah.py
import math

A = 5 
C = 12 
# pythagoras
B = int(math.sqrt(A ** 2 + C ** 2))
keliling = A + B + C
luas = int(1 / 2 * A * C)

print(f'Diketahui : \nAlas = {A} cm\nTinggi = {C} cm\n')
print(f'Jawab : \nSisi A = {C} cm\nSisi B = {B} cm\nSisi C = {A} cm')
print(f'Keliling = {keliling} cm\nLuas = {luas} cm²')